ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/transactions/TransactionPool.java

```diff
@@ -80,6 +91,9 @@ public TransactionPool(
   public void addRemoteTransactions(final Collection<Transaction> transactions) {
     final Set<Transaction> addedTransactions = new HashSet<>();
     for (final Transaction transaction : sortByNonce(transactions)) {
+      if (!syncState.isInSync()) {
```

Comment:
Looks like isInSync has most of the sync tolerance I was looking for already built in.  The acceptance criteria listed 100 blocks and the default is 5.  We cannot just change the default for isInSync as it is is also used by mining and 5 is their threshold.  

Can you update the isInSync to have a version that takes an argument and for this call use 100 (and make the no-args version call the args version with that constant)?  Either as a local constant or one of the new unstable options via SynchnizerConfiguration.

Reply:
Sounds good -- I made it a local constant 'cause I don't know how `SynchnizerConfiguration` works